commits_test.go

```diff
@@ -83,10 +86,18 @@ func TestSetCommitStatus(t *testing.T) {
 
 	mux.HandleFunc("/api/v4/projects/1/statuses/b0b3a907f41409829b307a28b82fdbd552ee5a27", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "POST")
+		body, err := ioutil.ReadAll(r.Body)
+		require.NoError(t, err)
+		var content SetCommitStatusOptions
+		err = json.Unmarshal(body, &content)
+		require.NoError(t, err)
+		assert.Equal(t, "ci/jenkins", *content.Name)
+		assert.Equal(t, 99.9, *content.Coverage)
 		fmt.Fprint(w, `{"id":1}`)
 	})
 
-	opt := &SetCommitStatusOptions{State: Running, Ref: String("master"), Name: String("ci/jenkins"), Context: String(""), TargetURL: String("http://abc"), Description: String("build")}
+	cov := 99.9
+	opt := &SetCommitStatusOptions{State: Running, Ref: String("master"), Name: String("ci/jenkins"), Context: String(""), TargetURL: String("http://abc"), Description: String("build"), Coverage: &cov}
```

Comment:
I notice it already was this long, but please keep the line length below 100 chars for lines of code (80 chars for documentation). So if you could fix this one that would be great.